spec/support/spec_test_helper.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# spec/support/spec_test_helper.rb
+module SpecTestHelper
+  def login_admin
+    login(:admin)
+  end
+
+  def login(user)
+    user = User.where(:nick => user.nick).first if user.is_a?(Symbol)
```

Comment:
why not `User.find_by_nick`?
i'd prefer `String` over `Symbol`.
AFAIK a `Symbol` does not support `.nick`